assignment4.Rmd

```diff
@@ -89,8 +90,12 @@ D4 <- cbind(D3, as.data.frame(D1$mean_correct))
 
 #Now re-run your scatterplots and correlations between the transformed data and mean_correct. If you had dropped some components would you have lost important infomation about mean_correct?
 
-COR2 <- 
-
+COR2 <- cor(D4)
+corrplot(COR2, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+#I won't lost important information about mean_correct because PC3 and PC5 have little impact on mean_correct (correlations=1).
```

Comment:
But what about PC6 that you dropped? I think if you check the relationship between PC6 and mean_correct you will find a strong relationship. The overall point is that you have to be careful when you use this method to drop variables as you can throw away important variation even if you are throwing away very little.